packages/config/src/test/discovery/config.test.ts

```diff
@@ -247,4 +249,28 @@ describe('discovery config.jsonc', () => {
       }
     }
   })
+
+  it('model-permissions is up to date', () => {
```

Comment:
On my laptop this test takes around 1-1.5s. It might become a problem on the CI, there probably are ways to speed it up. But that's for later, just something to be aware of.

For safety we might want to add `.timeout(5000)` to this test, just to give ourselves some time